tdrs-backend/plg/grafana/dashboards/logs_dashboard.json

```diff
@@ -23,6 +23,94 @@
   "panels": [
     {
       "datasource": {
+        "default": true,
```

Comment:
I know it is duplicative, but you need to also make the same changes on the `dev_logs_dashboard`. That is the dashboard for logs that will be visible for every user who is not an admin or OFA user. This was a request by Alex to keep the prod logs from being viewed outside of OFA. We need two dashboards because of the limited permissions OSS Grafana provides.